relayer/provider/provider.go

```diff
@@ -31,6 +31,7 @@ type ProviderConfig interface {
 	NewProvider(log *zap.Logger, homepath string, debug bool, chainName string) (ChainProvider, error)
 	Validate() error
 	BroadcastMode() BroadcastMode
+	BroadcastMaxMsgNum() uint64
```

Comment:
can we just name this `BroadcastMaxMsg`? the suffix of `Num` feels redundant given the type is an integer and dropping the suffix is a bit less verbose

Reply:
actually it's to avoid confusion if there's need for BroadcastMaxMsgBytes later

Reply:
that seems fair to me but perhaps it is a premature optimization, i guess i'm not strongly opinionated on this.
@Reecepbcups any thoughts on this?